tests/ui/feature-gates/unknown-feature.stderr

```diff
@@ -1,9 +1,33 @@
 error[E0635]: unknown feature `unknown_rust_feature`
-  --> $DIR/unknown-feature.rs:1:12
+  --> $DIR/unknown-feature.rs:2:5
    |
-LL | #![feature(unknown_rust_feature)]
-   |            ^^^^^^^^^^^^^^^^^^^^
+LL |     unknown_rust_feature,
+   |     ^^^^^^^^^^^^^^^^^^^^
 
-error: aborting due to 1 previous error
+error[E0635]: unknown feature `associated_types_default`
```

Comment:
do I see correctly here that it used to only error on the first unknown feature, and now on all?